nautobot/core/templatetags/helpers.py

```diff
@@ -616,6 +616,51 @@ def render_uptime(seconds):
     )
 
 
+@library.filter()
+@register.filter()
+def hyperlinked_field(value, hyperlink=None):
+    """Render a value as a hyperlink."""
+    if not value:
+        return placeholder(value)
+    hyperlink = hyperlink or value
+    return format_html('<a href="{}">{}</a>', hyperlink, value)
+
+
+@library.filter()
+@register.filter()
+def render_content_types(value):
+    """Render sorted by model and app_label ContentTypes value"""
+    if value.exists() is False:
+        return HTML_NONE
+    sorted_content_types = sorted(value.all(), key=lambda ct: (ct.model, ct.app_label))
```

Comment:
Huh. Is it intentional to sort by model first rather than by app label? That seems counterintuitive to me.

Also, would be more efficient I think to do `value.order_by()` instead of sorting the queryset in code?